src/Components/GrowthChart/ChartSettingsButton/PopoverList/PopoverListItem.tsx

```diff
@@ -0,0 +1,16 @@
+import React from 'react';
+
+type PopoverListItemProps = {
+  label: string;
+  icon: React.ReactNode;
+  onClick: () => void;
+};
+
+export const PopoverListItem = ({ label, icon, onClick }: PopoverListItemProps): JSX.Element => (
+  <div 
+  className="flex flex-row cursor-pointer hover:bg-gray-100 py-1 px-2 rounded min-w-fit"
```

Comment:
dosnt look like this hover:bg-gray-100 is working, would be nice to have the background change on hover